recipes/cytoolz/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.7.5" %}
+
+package:
+  name: cytoolz
+  version: {{ version }}
+
+source:
+  fn: cytoolz-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/c/cytoolz/cytoolz-{{ version }}.tar.gz
+  md5: 15fbc1c56fca80cbca0e89f37c93d432
+
+build:
+  number: 0
+  script: python setup.py install --record=record.txt
+
+requirements:
+  build:
+    - setuptools
+    - python
+    - cython
+    - toolz
+    - libpython  # [win and not (win64 and py34)]
```

Comment:
Yep, looking at the PR, it was added essentially to add a compiler, because presumably @cpcloud didn't have the right compiler (vs 2008?) installed, and perhaps conda-build's compiler finding logic also wasn't as good as it is now.